### PR TITLE
Enhanced the Login and Signup UI

### LINKED ISSUE #2374: Enhancing the LoginActivity and SignupActivity UI
**Actual Behaviour**

Lot of unused space is there and the UI can be improved by the implementation of card view.

**Expected Behaviour**

CardView can be implemented to enhance the UI. 

**Steps to reproduce it**

Navigate to the LoginActivity and SignUp Activity through the Navigation Drawer.

**Would you like to work on the issue?**

YES

### PR DESCRIPTION
**Pull request by akhil117**
Fixes #2374

Changes: Enhanced the Login and Signup UI by adding card view.

Screenshots for the change: 
screenshot_2018-03-14-01-11-19
screenshot_2018-03-14-01-11-23

### CODE REVIEW

**Review by Hiteshgautam01 — COMMENTED** [review]
Add all the values to dimens.xml

**Review by Hiteshgautam01 — APPROVED** [review]

File: android/app/src/main/res/layout/content_login.xml [hunk 1]
```diff
@@ -2,12 +2,21 @@
...
     app:layout_behavior="@string/appbar_scrolling_view_behavior">
-
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_marginTop="30dp"
+        android:layout_height="400dp"
+        android:layout_gravity="center"
+        android:layout_marginBottom="20dp"
+        android:background="@color/feed_bg"
+        card_view:cardCornerRadius="12dp"
```
Comment (Hiteshgautam01) [inline android/app/src/main/res/layout/content_login.xml:18]: Decrease the the value of the corner radius

File: android/app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
     <include layout="@layout/content_login" />
 
+
```
Comment (Hiteshgautam01) [inline android/app/src/main/res/layout/activity_login.xml:27]: Why are you adding unnecessary lines?

**Review by bhaveshAn — APPROVED** [review]

File: android/app/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#Tue Mar 13 00:07:52 IST 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (nikit19) [inline android/app/gradle/wrapper/gradle-wrapper.properties:6]: this

File: android/app/gradlew [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+fi
+
+# Split up the JVM_OPTS And GRADLE_OPTS values into an array, following the shell quoting and substitution rules
+function splitJvmOpts() {
+    JVM_OPTS=("$@")
+}
+eval splitJvmOpts $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS
+JVM_OPTS[${#JVM_OPTS[*]}]="-Dorg.gradle.appname=$APP_BASE_NAME"
+
+exec "$JAVACMD" "${JVM_OPTS[@]}" -classpath "$CLASSPATH" org.gradle.wrapper.GradleWrapperMain "$@"
```
Comment (nikit19) [inline android/app/gradlew:160]: this

File: android/app/gradlew.bat [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+:fail
+rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
+rem the _cmd.exe /c_ return code!
+if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
+exit /b 1
+
+:mainEnd
+if "%OS%"=="Windows_NT" endlocal
+
+:omega
```
Comment (nikit19) [inline android/app/gradlew.bat:90]: this

File: android/app/local.properties [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Tue Mar 13 00:07:49 IST 2018
+sdk.dir=/home/akhilbatchu/Android/Sdk
```
Comment (nikit19) [inline android/app/local.properties:11]: this

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please remove these files. They are not needed

**Comment by iamareebjamal** [comment]
IMO this is not needed

**Pull request closed without merging**